getnacelle/nacelle-js
# feat(ENG-6877): return nacelleEntryId and sourceEntryId with cart

### WHY are these changes introduced?

Addresses [ENG-6877](https://nacelle.atlassian.net/browse/ENG-6877)

> Return `nacelleEntryId` and `sourceEntryId` with cart contents



### WHAT is this pull request doing?



- Updates fragments to include new Nacelle values
- Creates new types to accommodate new Nacelle keys
- Creates utility function `transformShopifyLineToNacelleLine`
- Updates mocks where needed
- Updates tests where needed

## 

### How to Test

1. Check out the `ENG-6887/cart-return-nacelleEntryIds` branch.
2. `npm run bootstrap` from the monorepo root, as needed.
3. `cd packages/shopify-cart`
4. `npm run test:ci` - tests should pass with expected coverage
<img width="342" alt="Screen Shot 2022-09-14 at 10 18 13 AM" src="https://user-images.githubusercontent.com/24395931/190179849-33811c67-1b57-4d8e-86ec-792ec4e97247.png">
